src/Mailgun/Api/MailingList.php

```diff
@@ -254,7 +254,15 @@ public function addMembers($list, array $members, $upsert = 'no')
     {
         Assert::stringNotEmpty($list);
         Assert::isArray($members);
-        Assert::maxCount($members, 1000);
```

Comment:
Maybe you can try with this: `Assert::allMaxCount($values, $max, $message = '')`

https://github.com/webmozart/assert/blob/master/src/Assert.php#L159

Reply:
i think 6 lines of code don't hurt until the minimum required version of webmozart/assert get's uped to 1.3 or something like that. afaik it's the only method which requires such validation on the max array length